go-bindata-assetfs/main.go

```diff
@@ -29,6 +29,13 @@ func isDebug(args []string) bool {
 }
 
 func main() {
+	debug := isDebug(os.Args[1:])
```

Comment:
That makes sense but I quickly lost my interest in the repo after finding some other bugs and the lack of any test.